main.py

```diff
@@ -109,23 +115,21 @@ def load_chunks(inventory, lg, ck):
   return chks
 
 
-def load_vectors(docs):
+def load_vector(d):
```

Comment:
Is it necessary to change the function signature here (from plural to singular and docs to d)?

Reply:
I changed the function name to the singular because it better describes the new behavior of the function, but changing the argument name was not necessary